rosa_planning/rosa-sts-aws-prereqs.adoc

```diff
@@ -29,7 +29,7 @@ include::modules/rosa-sts-aws-requirements-account.adoc[leveloffset=+2]
 [id="additional-resources_aws-account-requirements_{context}"]
 .Additional resources
 * xref:../rosa_planning/rosa-limits-scalability.adoc#rosa-limits-scalability[Limits and scalability]
-* xref:../sd_support/rosa-troubleshooting-deployments.adoc#rosa-troubleshooting-general-deployment-elb[Creating the Elastic Load Balancing (ELB) service-linked role]
+* xref:../sd_support/rosa-troubleshooting-deployments.adoc#rosa-troubleshooting-elb-service-role[Creating the Elastic Load Balancing (ELB) service-linked role]
```

Comment:
I tested this in the preview, and it doesn't jump right down to the "Creating the Elastic Load Balancing (ELB) service-linked role" section. It stays at the top of the page. 

It looks like it's missing the context piece of the anchor ID. Based on what the section shows when you click on it, I _think_ it would be:

```suggestion
* xref:../sd_support/rosa-troubleshooting-deployments.adoc#rosa-troubleshooting-elb-service-role_rosa-troubleshooting-cluster-deployments[Creating the Elastic Load Balancing (ELB) service-linked role]
```

But I didn't actually test. 

Same for the xref in the other file.

Reply:
Thanks @bergerhoffer. Just tested and that is indeed the link it should have gone to.